huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go

```diff
@@ -351,6 +354,15 @@ func resourceFgsFunctionV2Create(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	if tagList, ok := d.GetOk("tags"); ok {
+		opts := function.TagsActionOpts{
+			Tags: utils.ExpandResourceTags(tagList.(map[string]interface{})),
+		}
+		if err := function.CreateResourceTags(fgsClient, d.Id(), opts); err != nil {
+			log.Printf("[ERROR] failed to add tags to FunctionGraph function (%s): %s", d.Id(), err)
```

Comment:
The create error should return `diag.Errorf("....")`

Reply:
some region does not support this function.

Reply:
How about add some description for this:
// Some region not support Funtion Tags, ignore errors.

Reply:
emmm, throw the error is right, if corresponding region is not support tags parameter, this parameter should not be used int the script.